packages/now-cli/test/integration.js

```diff
@@ -519,6 +519,27 @@ test('Deploy `api-env` fixture and test `now env` command', async t => {
   await nowEnvLsIsEmpty();
 });
 
+test('deploy with metadata containing "=" in the value', async t => {
+  const target = fixture('redirects-v2');
+
+  const { exitCode, stderr, stdout } = await execa(
+    binaryPath,
+    [target, ...defaultArgs, '--confirm', '--meta', 'someKey=='],
+    { reject: false }
+  );
+
+  t.is(exitCode, 0, formatOutput({ stderr, stdout }));
+
+  const url = stdout.split('https://'.length);
```

Comment:
```
> 'https://asdfsadf.com'.split('https://'.length)
[ 'https://asdfsadf.com' ]

> 'https://asdfsadf.com'.substring('https://'.length)
'asdfsadf.com'
```

I don't think you want `.split()`, but instead `.substring()`? 😆

```suggestion
  const url = stdout.substring('https://'.length);
```

Reply:
Seems like this logic was copy/pasted from another test. Can you fix it there as well? I think the fact that it works is an accident (since 1. the API supports `https://` prefix and 2. the `split()` output array contains only one element, so `String(url)` just turns back into `url`).

Reply:
🤦‍♂️

Reply:
Yes!

Reply:
I wrote "split" by mistake but it was "slice" in other tests. Still, we don't need to slice anything, so I'll update the other tests with `url = stdout` 👍

Reply:
(Only one test was wrong, updated here: https://github.com/zeit/now/pull/4160/commits/24301d11f4c8cdaaaaec621a18d077f271a9a87c)